pnc-core/src/main/java/org/jboss/pnc/core/BuildDriverFactory.java

```diff
@@ -25,7 +25,7 @@ public BuildDriver getBuildDriver(BuildType buildType) throws CoreException {
             }
         }
 
-        throw new CoreException("No build driver available for " + buildType + " build type.");
+        throw new CoreException("No build driver available for " + buildType + " build type."); //TODO create test to make sure exception is properly handled
```

Comment:
Why do you think it's wrong if TODO is left there, we are not GA yet? Before removing it we need to make sure that it is done. Do you think we should create jira for it? I think Jira ticket has to much overhead for such task.